html/cases.php

```diff
@@ -38,7 +38,8 @@ function get_by_priority_id($id) {
 		}
 
 		// connect
-	$dbh = new PDO('mysql:host=localhost;dbname=masmallc_scas', 'masmallc_scas', "LWn-tmX-ETv-N7M");
+	$dbh = new PDO("mysql:dbname=" . QUERY_DATABASE . ";host=" . QUERY_SERVER, QUERY_USERNAME, QUERY_PASSWORD);
+	// $dbh = new PDO('mysql:host=localhost;dbname=masmallc_scas', 'masmallc_scas', "LWn-tmX-ETv-N7M");
```

Comment:
do you mind removing the commented out line? thank you!!

Reply:
done!